### PR TITLE
Test against ember.js 1.10 - 1.13 in CI

### PR DESCRIPTION
**Pull request by mike-north**
* Delete coverage info from version control, and gitignore
* Remove unnecessary ember-data npm dependency
* Add ember-try scenarios for 1.10, 1.11, 1.12, 1.13 framework versions

### CODE REVIEW

**Comment by offirgolan** [comment]
:+1:  Looks good! Thanks @mike-north

File: package.json [hunk 1]
```diff
@@ -41,7 +41,6 @@
     "ember-cli-moment-shim": "0.6.1",
     "ember-cli-qunit": "1.0.1",
     "ember-cli-uglify": "^1.0.1",
-    "ember-data": "~2.0.0",
```
Comment (blimmer) [inline package.json]: Don't we need this for our dummy app tests?

File: .travis.yml [hunk 2]
```diff
@@ -14,7 +14,10 @@ addons:
     repo_token: 32e4df25574ba02aacb75e6d34b9c667044984df279e20a208884d7e9e9d4a8d
 
 env:
-  - EMBER_TRY_SCENARIO=default
+  - EMBER_TRY_SCENARIO=ember-1.10
+  - EMBER_TRY_SCENARIO=ember-1.11
+  - EMBER_TRY_SCENARIO=ember-1.12
```
Comment (blimmer) [inline .travis.yml]: Do we want to consider some of these (1.10) allowed failures?

File: package.json (hunk 1 above)
Comment (offirgolan) [inline package.json]: Yes we do. Idk how I missed that

File: .travis.yml (hunk 2 above)
Comment (mike-north) [inline .travis.yml]: why?

File: package.json (hunk 1 above)
Comment (mike-north) [inline package.json]: Why? There's already a bower reference to ember-data

Comment (offirgolan) [inline package.json]: I think as of right now its needed in both package.json and bower.json. This change broke the demo app so I had to revert it.

Comment (rwjblue) [inline package.json]: We should probably add a handful of acceptance tests to ensure the dummy app still functions...

Comment (rwjblue) [inline package.json]: Yes, ED must be in both package.json and bower.json to be included.

**Pull request merged**